### ISSUE TITLE
73

### ISSUE THREAD

**Issue by vaskonov** [deep-nlp-spring-2020/Task-1-VMAtm#1]
Please do not hesitate to contact me if you've noticed some inexactness here.

**Assignment 1.1: 23/25** 

**Question 2.3**
-1: not precise explanation. Sysonyms and antonyms often occur in the similar contexes that's why sometimes antonym is closer to a word than it's synonym.

**Question 2.7 - 2.8**
-1: not precise explanation. In short, there is a bias in text that transfers into the word embeddings. More detailed explanation can be found [here](https://towardsdatascience.com/gender-bias-word-embeddings-76d9806a0e17)

**Assignment 1.2: 20/20**

**Assignment 1.3: 30/40**

- non canonical CBOW implementation. Please take a look at the [word2vec Parameter Learning Explained](https://arxiv.org/abs/1411.2738). Some details:
- non-linearity should not be used here.
- `x.mean(dim=1)` should be either before `nn.Embedding` or after it (formulas 17,18), but not in the output layer.
Very nice analysis, especially plot with colours and the catch with `gensim.evaluate_word_analogies` 

**Assignment 1.4: 0/15**
